test/spec/requestBuilder.js

```diff
@@ -5,7 +5,35 @@ var test = require('tape');
 var requestBuilder = require('../../src/requestBuilder.js');
 var getQueries = requestBuilder._getQueries;
 
-test('The request builder should generate queries according to the search parameters', function(t) {
+test('The request builder should set analytics to subsequent queries', function(t) {
```

Comment:
Just a question, what is the logic of subsequent queries? I'm not sure to understand why the second query should have the param analytics false?

Reply:
That's to make sure that if a user types one letter, it's not counted as multiple queries, for example in hierarchical faceting